core/src/main/java/bisq/core/btc/setup/MoneroWalletRpcManager.java

```diff
@@ -48,8 +50,9 @@ public MoneroWalletRpc startInstance(List<String> cmd) {
     try {
```

Comment:
could this mehtod be called multiple times (with the same configured port)?

Reply:
Yes, it's possible, which would cause an error binding to the same port. Until now the ports were auto assigned.

Reply:
i will have a look why is that happening but is that correct to start two instances (I could add +1 to the port or something) or should I try to reuse the old one if it was started already? (I will see what the other params are when starting it)